doc/NHibernate.shfbproj.template

```diff
@@ -23,7 +23,7 @@
     <FeedbackEMailLinkText>nhibernate team</FeedbackEMailLinkText>
     <HelpTitle>NHibernate API Reference</HelpTitle>
     <IndentHtml>False</IndentHtml>
-    <HelpFileVersion>${project.version.numeric}</HelpFileVersion>
+    <HelpFileVersion>${project.version}</HelpFileVersion>
```

Comment:
What does this change? It seems to me this help file version is hard to see anywhere. I have generated the api chm just to check, but I have not seen it anywhere when opening the file or checking its properties.

Reply:
It's not about documentation. See discussion here https://github.com/nhibernate/nhibernate-core/pull/2254#discussion_r336746720. If I remember correctly it generates specification for API - takes methods declarations, comments from code and generates html from it. Something like it. It's not part of the build now.

Reply:
Ok I have forgot about that property removal, and replacing it here is fine.

> I have generated the api chm

I know what this SHFB file is about. It is for generating the API chm, and that is what I have done to seek where this version was appearing. (Run `nant api` into `doc` folder. It takes a very long time.) So far I have not found it, but well, I now have the reason for the change, so it is ok for me.

Yes

Reply:
Can you share it? Interesting to take a look bus was lazy to install some required apps.

Reply:
For some reason, the upload fails on GitHub. SO I have used some sharing service, which delete the file after 30 days without download. It is [here](http://dl.free.fr/ohxC4Q4SK).

Reply:
Thanks!